specification/professionalservice/resource-manager/Microsoft.ProfessionalService/preview/2023-07-01-preview/professionalservice.json

```diff
@@ -0,0 +1,1038 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProfessionalService/resources/{resourceName}/unsubscribe": {
+      "post": {
+        "tags": [
+          "SubscriptionLevelProfessionalServiceResources"
+        ],
+        "operationId": "ProfessionalServiceSubscriptionLevel_UpdateToUnsubscribed",
+        "description": "Unsubscribe from a specified Subscription Level ProfessionalService.",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/deleteOptions"
+            },
+            "description": "Parameters supplied to unsubscribe ProfessionalService operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Unsubscribe operation completed successfully."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "description": "The URL to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "format": "int32",
+                "type": "integer"
+              }
+            }
+          },
+          "204": {
+            "description": "Unsubscribe operation completed successfully."
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Unsubscribe Subscription Level ProfessionalService resource": {
+            "$ref": "./examples/ProfessionalService/ProfessionalServiceUnsubscribe.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources": {
```

Comment:
"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources"
no provider in the path?